app/views/layouts/application.html.erb

```diff
@@ -6,7 +6,9 @@
   <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
   <%= csrf_meta_tags %>
 
-  <%= favicon_link_tag 'favicon.png', type: 'image/png' %>
+  <%= favicon_link_tag 'images/favicon.png', type: 'image/png' %>
+  <%= favicon_link_tag 'images/stories-icon-apple-touch-180x180.png', rel: 'apple-touch-icon', type: 'image/png' %>
+
```

Comment:
Can you take out 'images/' from the strings?  Rails will find the image automatically.